contents/newsletter/product-pricing-advice.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Non-obvious advice about pricing your product
+date: 2025-10-15
+author:
+ - ian-vanagas
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/bookworm_fccbcec132.png
+featuredImageType: full
+tags:
+  - Product
+  - Founders
+crosspost:
+  - Founders
+  - Blog
+---
+
+We care a lot about pricing at PostHog. As proof:
+
+- Nearly [all of our products](/products) have separate pricing, meaning we are always figuring it out for new products or adjusting it for existing ones.
+
+- [Enduringly low prices](/handbook/low-prices) is a core value of PostHog, which our customers love.
+
+- There are over [100 RFCs](/newsletter/how-we-work-async#5-our-request-for-comments-process) (108 to be exact) about pricing in our private (sorry) product discussions repo.
+
+All of this has meant we’ve thought, debated, and learned a lot about pricing. We’re sharing the best bits we’ve learned here. 
+
+## 1. How you price is who you are
+
+Pricing is often relegated to an afterthought when building a product, but it shouldn't be. It's as impactful to the overall product experience as the functionality or design.
+
+To show you how, imagine two [feature flag](/newsletter/feature-flag-mistakes) tools.
+
+The first charges by **seats** and **flags created**. 
+
+Their customers think about every flag and user they add, meaning they do less of both. The flags they do create are more important and are used more heavily. This model appeals to larger companies with more requests per flag, and that means enterprise features like auditability and access controls are more important.
+
+The second charges by **requests**. 
+
+This means more developers can access and create flags. There are more flags created, but where they put those flags becomes more important so they aren’t charged too much for requests. This appeals to smaller companies who ship more flags, so cost controls and DevEx features like local evaluation matter more.
+
+These two examples show **how much** you charge often doesn’t matter as much as **how** you charge. There are many ways this can vary like:
+
+- Usage-based pricing vs. seat-based pricing.
+- Pay-as-you-go vs. annual contracts vs. licenses.
+- Self-serve vs. custom contracts with sales.
+
+Deciding your pricing model comes down to your [ideal customer](/newsletter/ideal-customer-profile-framework). For us, that is [product engineers](/blog/what-is-a-product-engineer) at high-growth startups. The ability to self-serve and have pricing align with usage is important for them, so that is what we’ve gone with.
```

Comment:
```suggestion
Ultimately, how you price your product should stream from your [ideal customer profile](/newsletter/ideal-customer-profile-framework) (ICP). Our ICP is high-growth startups that are led by [product engineers](/blog/what-is-a-product-engineer), so the ability self-serve and have pricing align with usage is important for them, so that is what we’ve gone with.
```